docker-compose-example.yml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2016 Capital One Services, LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+version: '2'
```

Comment:
Based on the docs in the README, I was expecting `docker-compose-example.yml` to be able to build the commander, as well as the example processor. Does this do that?